pvlib/iotools/era5.py

```diff
@@ -0,0 +1,276 @@
+        Type of data object to return. Default is to return a pandas DataFrame
+        if file only contains one location and otherwise return an xarray
+        Dataset.
+    map_variables: bool, default: True
+        When true, renames columns of the DataFrame to pvlib variable names
+        where applicable. See variable ERA5_VARIABLE_MAP.
+
+    Notes
+    -----
+    The returned data includes the following fields by default:
+
+    ========================  ======  =========================================
+    Key, mapped key           Format  Description
+    ========================  ======  =========================================
+    *Mapped field names are returned when the map_variables argument is True*
+    ---------------------------------------------------------------------------
+    2tm, temp_air             float   Air temperature at 2 m above ground [K]
+    u10                       float   Horizontal airspeed towards east at 10 m [m/s]
+    v10                       float   Horizontal airspeed towards north at 10 m [m/s]
+    sp, pressure              float   Atmospheric pressure at the ground [Pa]
+    msdwswrf, ghi             float   Mean surface downward short-wave radiation flux [W/m^2]
+    msdwswrfcs, ghi_clear     float   Mean surface downward short-wave radiation flux, clear sky [W/m^2]
+    msdrswrf, bhi             float   Mean surface direct short-wave radiation flux [W/m^2]
+    msdrswrfcs, bhi_clear     float   Mean surface direct short-wave radiation flux, clear sky [W/m^2]
+    ========================  ======  =========================================
+
+    Returns
+    -------
+    data: DataFrame
+        ERA5 time-series data, fields depend on the requested data. The
+        returned object is either a pandas DataFrame or an xarray dataset,
+        depending on the output_format parameter.
+    metadata: dict
+        Metadata for the time-series.
+
+    See Also
+    --------
+    pvlib.iotools.read_era5
+
+    References
+    ----------
+    .. [1] `ERA5 hourly data on single levels from 1979 to present
+       <https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=overview>`_
+    .. [2] `ERA5 data documentation
+       <https://confluence.ecmwf.int/display/CKB/ERA5%3A+data+documentation>`_
+    .. [3] `How to use the CDS API
+       <https://cds.climate.copernicus.eu/api-how-to>`_
+    .. [4] `CDSAPI source code
+       <https://github.com/ecmwf/cdsapi>`_
+    .. [5] `Climate Data Store (CDS) API Keywords
+       <https://confluence.ecmwf.int/display/CKB/Climate+Data+Store+%28CDS%29+API+Keywords>`_
+    .. [6] `Climate Data Storage user registration
+       <https://cds.climate.copernicus.eu/user/register>`_
+    """  # noqa: E501
+    if cdsapi is None:
+        raise ImportError('Retrieving ERA5 data requires cdsapi to be installed.')  # noqa: E501
+
+    cds_client = cdsapi.Client(url=CDSAPI_URL, key=api_key, verify=1)
```

Comment:
You don't think this output is useful for the user?

I've set `quiet=True`.

Reply:
from https://pvlib-python.readthedocs.io/en/latest/contributing.html#code-style:

> Remove any logging calls and print statements that you added during development.

I guess it's not 100% relevant since you didn't add the logging yourself, but I think the point is that logging should be disabled by default